dashboard/src/actions/charts.ts

```diff
@@ -156,3 +166,18 @@ export function getChartValues(
     }
   };
 }
+
+export function getChartUpdates(
+  name: string,
+  version: string,
+  appVersion: string,
+): ThunkAction<Promise<void>, IStoreState, null, ChartsAction> {
+  return async dispatch => {
+    try {
+      const chartUpdates = await Chart.checkUpdates(name, version, appVersion);
+      dispatch(receiveChartUpdates(chartUpdates));
```

Comment:
IMO this action should just extract the newer version (if there is one) and the repo(s) it's from and send that info to the reducer. I don't think we need to store the full filtered charts response in the state and send that to AppView.